logistics_planning_base/views/logistics_schedule_views.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="logistics_schedule_view_tree" model="ir.ui.view">
+        <field name="name">logistics.schedule.tree</field>
+        <field name="model">logistics.schedule</field>
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <button
+                    name="action_logistics_schedule_form_view"
+                    icon="fa-gear"
+                    class="p-0"
+                    type="object"
+                    title="Edit"
+                />
+                <field name="partner_id" />
+                <field name="stock_move_id" domain="[('partner_id', '=', partner_id), ('product_id', '=', product_id), ('state', '=', 'done'), ('logistics_schedule_id', '=', False)]"/>
+                <field name="transport_type" />
+                <field name="product_id" />
+                <field name="product_uom_qty" />
+                <field name="scheduled_load_date" />
+                <field name="commitment_date" />
+                <field name="currency_id" invisible="1"/>
+                <field name="logistic_price_unit" widget="monetary" />
+                <field name="logistic_price_unit_done" widget="monetary" />
+                <field name="carrier_id" />
+                <field name="license_plate_1" />
+                <field name="license_plate_2" />
+                <field name="state" readonly="1"/>
+                <field name="note" />
+                <button
+                    name="%(action_logistics_schedule_cancel)d"
+                    icon="fa-ban"
+                    class="p-0"
+                    type="action"
+                    title="Cancel"
```

Comment:
Do you remember if it was possible adding a `confirm` attribute that enables a warning message before completing such action?

Reply:
I have done several tests and it seems that in the tree view the confirm attribute does not work, instead in the form, for example, i can make a simple wizard to confirm this action, would you prefer?

Reply:
OK, temporary hide cancel button in `tree` view. And add `confirm` in `form` button view.